addons/web/static/src/js/views/abstract_controller.js

```diff
@@ -119,9 +118,9 @@ var AbstractController = mvc.Controller.extend(ActionMixin, {
      */
     on_attach_callback: function () {
         ActionMixin.on_attach_callback.call(this);
+        this.searchModel.on('search', this, this._onSearch);
         if (this.withControlPanel) {
```

Comment:
maybe add 'this.searchModel &&'?

Reply:
Not needed now that all actions have a searchModel